functions/_common/explorerHandlers.ts

```diff
@@ -216,28 +216,26 @@ export async function fetchZipForExplorerView(
 
     try {
         const explorerEnv = stripUrlExtensionFromEnv(env, extensions.zip)
-        const { grapherState, explorerParams } =
-            await initGrapherForExplorerView(explorerEnv, options)
+        const { grapherState } = await initGrapherForExplorerView(
+            explorerEnv,
+            options
+        )
 
         ensureDownloadOfDataAllowed(grapherState)
         const metadata = assembleMetadata(grapherState, searchParams)
         const readme = assembleReadme(grapherState, searchParams)
         const csv = assembleCsv(grapherState, searchParams)
         console.log("Fetched the parts, creating zip file")
 
-        // Make a unique identifier for the given view
+        // Extract explorer slug
         const explorerSlug = explorerEnv.url.pathname.split("/").pop()
-        const viewId = Object.values(explorerParams)
-            .map((value) => value.replace(/\s/g, "_"))
-            .join("__")
-        const identifier = `${explorerSlug}__${viewId}`
```

Comment:
The view id is ugly, so I prefer to just use the slug (we do the same for mdims)

Reply:
So if the user downloads data for multiple views, they will not have a way to distinguish between, right? Maybe we can come up with a nicer viewId (and add it also to multi-dims)?

Reply:
Yup, feel free to create an issue

Reply:
I think it would be best if we added a viewId column to the explorerViews table, but I'll do so in a separate PR